website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -1529,7 +1529,7 @@ $ curl \
 
 ## Unlock a workspace
 
-This endpoint unlocks a workspace.
+This endpoint unlocks a workspace. Unlocking a workspace will set the current state version if there are any provisional state versions available.
```

Comment:
will it set the current state version to the latest snapshot?

we might need to define "provisional state" in the docs (both here, and perhaps in workspace docs) so it's clear